src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/Rfc2898DeriveBytes.OneShot.cs

```diff
@@ -0,0 +1,134 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography
+{
+    public partial class Rfc2898DeriveBytes
+    {
+        private const int OneShotMinIterations = 1000;
+        private const int OneShotMinSaltLength = 128 / 8; // 128-bits
+        private const int OneShotMinExtractLength = 112 / 8; // 114-bits
```

Comment:
112, or 114?

Reply:
Ha. 112, which is 14 bytes, and I guess I just kind of smooshed together in the comment.

Reply:
Also, currently investigating: I think the minimum checks can go away. The way we are hitting OpenSSL, it might not be forcing the check.

Reply:
Yep, all the limits were able to be lifted. The salt length limitation is retained from the instance based one, though.